src/main/java/app/bpartners/api/endpoint/event/model/InvoiceExportLinkRequested.java

```diff
@@ -0,0 +1,37 @@
+package app.bpartners.api.endpoint.event.model;
+
+import app.bpartners.api.endpoint.rest.model.ArchiveStatus;
+import app.bpartners.api.endpoint.rest.model.InvoiceStatus;
+import java.time.Duration;
+import java.time.LocalDate;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+@EqualsAndHashCode(callSuper = false)
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Data
+@ToString
+public class InvoiceExportLinkRequested extends PojaEvent {
```

Comment:
I think this event must be handle with EVENT_STACK_2 because it needs some memory support ... 

Just override getEventStack() { return EVENT_STACK_2;}